Source/WTF/wtf/PlatformUse.h

```diff
@@ -307,7 +307,7 @@
 
 #if !defined(USE_TZONE_MALLOC)
 #if (CPU(ARM64) || CPU(X86_64)) && OS(DARWIN) && (__SIZEOF_POINTER__ == 8)
-#define USE_TZONE_MALLOC 0 /* temporarily disable FIXME rdar://140325747 */
+#define USE_TZONE_MALLOC 1 /* temporarily disable FIXME rdar://140325747 */
```

Comment:
Is this intended to be 1? If so can we delete the radar?

Reply:
This was enabled just so we can get EWS testing with TZoneHeaps.  I've now uploaded a PR with this removed.